### ISSUE TITLE
correcting the spacing between words

### ISSUE THREAD

**Issue by MrZorG33** [We-the-People-civ4col-mod/Mod#471]
need to correct the indentation of the text, as for Port Royal. this fix is only required when trading with natives.
![image](https://user-images.githubusercontent.com/61798883/121247903-71d9e400-c8ab-11eb-9710-5c69ade60289.png)
![image](https://user-images.githubusercontent.com/61798883/121247936-79998880-c8ab-11eb-9efe-40f60f81c760.png)

**edit**
although, in my opinion, it would be better visually to reduce this interval.
![image](https://user-images.githubusercontent.com/61798883/121248581-44416a80-c8ac-11eb-878e-806506ae42ad.png)
![image](https://user-images.githubusercontent.com/61798883/121248655-5cb18500-c8ac-11eb-968e-431ff0061e4b.png)

doesn't look very nice ...

**Comment by kendonmcb** [comment]
Nope, wasn't me. I don't even know what this is.

**Comment by Nightinggale** [comment]
I just fixed it and then I noticed the edit. I didn't fix that part.

**Comment by raystuttgart** [comment]
I have no idea why these "indentations" in **Profession Mouse Over Help** were added ...
They are quite pointless and ugly ...

**Comment by Fuerstbischof** [comment]
re: Native trading screen

There is only one batch of tags covering sell/buy prices. The added tabs work perfectly on all other screens but on the Native trading screen it does actually work not. Of course it should, but it does not, which seems to hint there is something wrong with the python? screen...


re: indented text in tooltips of city screen

Readability is my prime concern. The second tab can be removed if the yield is no longer superficially added to lines listing the bonuses. Occasionally a bonus 100% or more shows which mad 2 tabs a necessity which is now no longer necessary.


re: rays rant

This is part of my ongoing project to improve the overall readability of pop-ups, tooltips etc. The original text is just sloppily coded and needed some polishing. Adding some tabs has fixed this abomination just fine. I'll upload the revised files tomorrow.

**Comment by Nightinggale** [comment]
> There is only one batch of tags covering sell/buy prices. The added tabs work perfectly on all other screens but on the Native trading screen it does actually work not. Of course it should, but it does not, which seems to hint there is something wrong with the python? screen...

I don't get why it isn't the same everywhere either. Maybe replace tabs with spaces will fix the problem.

> This is part of my ongoing project to improve the overall readability of pop-ups, tooltips etc.

Sounds good. Remember that we can modify the code for those (well most of those anyway) meaning we might be able to do stuff, which isn't available by only changing the text xml files. I'm not sure you would need that, but you should be aware of that option in case you run into limitations.

**Comment by raystuttgart** [comment]
@Fuerstbischof 
I was referring to this and it really looks ugly:
https://user-images.githubusercontent.com/61798883/121248581-44416a80-c8ac-11eb-878e-806506ae42ad.png

**Comment by Fuerstbischof** [comment]
> I don't get why it isn't the same everywhere either. Maybe replace tabs with spaces will fix the problem.

It should be the same but it isn't. If you've time to investigate - go ahead!
In the meantime I'll try fixing it with spaces if it's possible.

> > This is part of my ongoing project to improve the overall readability of pop-ups, tooltips etc.
> 
> Sounds good. Remember that we can modify the code for those (well most of those anyway) meaning we might be able to do stuff, which isn't available by only changing the text xml files. I'm not sure you would need that, but you should be aware of that option in case you run into limitations.

The most helpful change for tooltips would be in general a greater width. Word wrap would almost never happen then which would increase the readability, benefitting German texts mostly. Pop-ups could benefit from a greater width and a greater length. If the texts of pop-ups could support different font sizes that would be beneficial, too.

**Comment by Fuerstbischof** [comment]
> Ok, I have seen now the rest, with all the other modifiers and data in the **"Profession Mouse Over Help".**, I understand.
> And yes it is a lot cleaner now. (The first Screenshots of @Mr.Zorg simply gave me a different impressions.)
> 
> The rest - as shown in your screenshots - also looks perfectly fine for me. :)
> 
> So sorry. 👍

Apology accepted.

You'd know me by now. I'm doing things for a reason. ;-)

**Comment by Fuerstbischof** [comment]
I was just to upload the revised file with now works for all languages but sadly it can't make it into 3.0 whose release has been moved forward for unknown reasons.

I've fixed the spacing issue by inserting additional spaces followed by a tab. The problem originated from the different size of the fonts used by the tooltips to display the text. As can be easily seen the size of a tab is different...
![WTP-font sizes](https://user-images.githubusercontent.com/45920798/122667282-4d86dd00-d1b2-11eb-88d3-9a1b9cd88f06.png)
